Gruntfile.js

```diff
@@ -18,7 +18,10 @@ module.exports = function(grunt) {
                 wptApiKey: 'API_KEY_HERE',
                 hipchatApiKey: 'API_KEY_HERE',
                 roomId: 123456,
-                notifyHipchat: true
+                kibanaHost: 'localhost',
```

Comment:
Worth changing these parameters to "logstashHost" and "logstashPort" because it's really logstash support we're adding